HerPublicWebsite.ManagementShell/FakeReferralGenerator.cs

```diff
@@ -65,6 +63,7 @@ public IEnumerable<ReferralRequest> GenerateFakeReferralRequests(int count, int
             // > 36000 5%
             .RuleFor(rr => rr.IncomeBand,
                 f => f.Random.WeightedRandom(
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Would ideally just be around the obsolete ones

Reply:
done